### PR TITLE
Bugfix: Validator password and commands

### PR DESCRIPTION
**Pull request by Wolmin**
This PR presents changes for:
- validator commands
- logging error about wrong validator password

### CODE REVIEW

**Review by Hugoo — APPROVED** [review]
looks good

**Review by Hugoo — COMMENTED** [review]
image

2nd flag description is not correct

File: cmd/lukso/validator.go [hunk 1]
```diff
@@ -59,6 +63,40 @@ func startValidator(ctx *cli.Context) (err error) {
 		return cli.Exit(fmt.Sprintf("❌  There was an error while starting validator: %v", err), 1)
 	}
 
+	passwordPipe.Close()
+
+	log.Info("⚙️  Please wait a few seconds while we validate your password...")
```
Comment (Hugoo) [inline cmd/lukso/validator.go]:
```suggestion
	log.Info("⚙️  Please wait a few seconds while your password is being validated...")
```

File: cmd/lukso/validator.go [hunk 2]
```diff
@@ -59,6 +63,40 @@ func startValidator(ctx *cli.Context) (err error) {
...
+		if err != nil {
+			return cli.Exit(fmt.Sprintf("❌  There was an error while stopping consensus: %v", err), 1)
+		}
+
+		err = clientDependencies[cfg.Execution()].Stop()
+		if err != nil {
+			return cli.Exit(fmt.Sprintf("❌  There was an error while stopping execution: %v", err), 1)
+		}
+
+		return cli.Exit("❌  Password incorrect, please restart and try again", 1)
```
Comment (Hugoo) [inline cmd/lukso/validator.go]:
```suggestion
		return cli.Exit("❌  Incorrect password, please restart and try again", 1)
```

image

There are 2 emojis, i think it is too much. But lets keep it like this.

**Review by fhildeb — APPROVED** [review]

File: cmd/lukso/validator.go (hunk 1 above)
Comment (Wolmin) [reply cmd/lukso/validator.go]: Done

File: cmd/lukso/validator.go (hunk 2 above)
Comment (Wolmin) [reply cmd/lukso/validator.go]: Done

**Pull request merged**